pages/cloudflare/caching.mdx

```diff
@@ -15,24 +15,22 @@ Workers KV is eventually consistent, which means that it can take up to 60 secon
 
 ### How to enable caching
 
-`opennextjs/cloudflare` uses [Workers KV](https://developers.cloudflare.com/kv/) as the cache for your Next.js app. Workers KV is [fast](https://blog.cloudflare.com/faster-workers-kv) and uses Cloudflare's [Tiered Cache](https://developers.cloudflare.com/cache/how-to/tiered-cache/) to increase cache hit rates. When you write cached data to Workers KV, you write to storage that can be read by any Cloudflare location. This means your app can fetch data, cache it in KV, and then subsequent requests anywhere around the world can read from this cache.
+`@opennextjs/cloudflare` uses [Workers KV](https://developers.cloudflare.com/kv/) as the cache for your Next.js app. Workers KV is [fast](https://blog.cloudflare.com/faster-workers-kv) and uses Cloudflare's [Tiered Cache](https://developers.cloudflare.com/cache/how-to/tiered-cache/) to increase cache hit rates. When you write cached data to Workers KV, you write to storage that can be read by any Cloudflare location. This means your app can fetch data, cache it in KV, and then subsequent requests anywhere around the world can read from this cache.
 
 To enable caching, you must:
 
 #### 1. Create a KV namespace
 
 ```
-npx wrangler@latest kv namespace create NEXT_CACHE_WORKERS_KV
+npx wrangler@latest kv namespace create <YOUR_NAMESPACE_NAME>
 ```
 
 #### 2. Add the KV namespace to your Worker
 
+The binding name used in your app's worker will be `NEXT_CACHE_WORKERS_KV` by default. This is configurable and can be changed by setting the `__OPENNEXT_KV_BINDING_NAME` build-time environment variable.
```

Comment:
What about re-using `<YOUR_NAMESPACE_NAME>` and add a note about the default value at the bottom of this section.

Reply:
the binding name can differ from the namespace name (and maybe should if you have multiple next.js apps or preview vs prod), so i'm not sure that would be the right recommendation